src/components/Table/SortIndicator/SortIndicator.tsx

```diff
@@ -18,11 +18,11 @@ export const SortIndicator: React.FC<SortIndicatorProps> = ({
     );
 
     if (direction === 'descending') {
-        return <ArrowDown {...props} className={classNameProps}/>;
+        return <ArrowDown {...props} aria-label="arrowDown-icon" className={classNameProps}/>;
```

Comment:
As `aria-label` is for accessible name, you can be more descriptive: `Icon for sorting in descending order`